presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/session/PropertyMetadatas.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.base.session;
+
+import io.airlift.units.DataSize;
+import io.airlift.units.Duration;
+import io.prestosql.spi.session.PropertyMetadata;
+
+import static io.prestosql.spi.type.VarcharType.VARCHAR;
+
+public final class PropertyMetadatas
+{
+    private PropertyMetadatas() {}
+
+    public static PropertyMetadata<DataSize> dataSizeProperty(String name, String description, DataSize defaultValue, boolean hidden)
```

Comment:
There are a few of these for other types in https://github.com/prestosql/presto/blob/master/presto-spi/src/main/java/io/prestosql/spi/session/PropertyMetadata.java#L141 do you want to put these there instead?

Reply:
They are directly reusable, in SPI.

I can't put these methods in that place, because the methods use units and SPI doesn't have (and shouldnt have) dep on units.

Reply:
Ah, I see. Thanks.